.stylish-haskell.yaml

```diff
@@ -0,0 +1,177 @@
+# stylish-haskell configuration file
+# ==================================
```

Comment:
Please modify `README.md` in the Contributing section accordingly with a nano-tutorial on how to use stylish-haskell.